sumo/cli/phonon_bandplot.py

```diff
@@ -53,7 +53,7 @@ def phonon_bandplot(filename, poscar=None, prefix=None, directory=None,
                     symprec=0.01, mode='bradcrack', kpt_list=None,
                     eigenvectors=False, labels=None, height=6., width=6.,
                     ymin=None, ymax=None, image_format='pdf', dpi=400,
-                    plt=None, fonts=None):
+                    plt=None, fonts=None, dos_file=None):
```

Comment:
The option for combined DOS and electronic band structure plots is just `--dos`.

I think we should consider making them the same for consistency.

Reply:
done!